src/coreclr/gc/gc.cpp

```diff
@@ -3418,7 +3418,7 @@ gc_heap::dt_high_frag_p (gc_tuning_point tp,
                 }
                 dprintf (GTC_LOG, ("h%d: gen%d, frag is %zd, alloc effi: %d%%, unusable frag is %zd, ratio is %d",
                     heap_number, gen_number, dd_fragmentation (dd),
-                    (int)(100*generation_allocator_efficiency (generation_of (gen_number))),
+                    (int)(generation_allocator_efficiency_percent (generation_of (gen_number))),
```

Comment:
All places that call `generation_allocator_efficiency_percent` cast the result to int or uint32_t. Should the result be int or uint32_t to avoid these casts?

Reply:
Yes I can to that. I did not change it in previous commits because of flags in `printf` but I can do the change if you want. The result is always positive. Do you prefer `int` or `uint32_t` ?

Reply:
@Maoni0 Do you have a preference?